src/Address/StreetSuggester/BPostStreetSuggester.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Address\StreetSuggester;
+
+use GuzzleHttp\Psr7\Request;
+use GuzzleHttp\Psr7\Uri;
+use Psr\Http\Client\ClientInterface;
+
+final class BPostStreetSuggester implements StreetSuggester
+{
+    private ClientInterface $client;
+
+    private string $domain;
+
+    private string $token;
+
+    public function __construct(ClientInterface $client, string $domain, string $token)
+    {
+        $this->client = $client;
+        $this->domain = $domain;
+        $this->token = $token;
+    }
+
+    /**
+     * @return string[]
+     */
+    public function suggest(
+        string $postalCode,
+        string $locality,
+        string $streetQuery,
+        int $limit = 5
+    ): array {
+        $request = new Request(
+            'GET',
+            (new Uri($this->domain))
+                ->withPath('/roa-info-st2/externalMailingAddressProofingRest/autocomplete/street')
+                ->withQuery(http_build_query([
+                'id' => '7',
```

Comment:
I don't know this specific API and do not have the relevant documentation, but I wonder if this always needs to be hardcoded to `7` and if so why? Maybe a link to the API documentation and a comment explaining why we use this parameter could already be a big improvement here